{"address":"::","code":"EADDRINUSE","errno":-98,"level":"error","port":1338,"syscall":"listen","timestamp":"2023-06-13T11:24:39.370Z"}
{"code":105,"level":"error","message":"Invalid key name: _id","stack":"Error: Invalid key name: _id\n    at /home/sites/server/server/node_modules/parse-server/lib/Controllers/DatabaseController.js:130:13\n    at Array.forEach (<anonymous>)\n    at validateQuery (/home/sites/server/server/node_modules/parse-server/lib/Controllers/DatabaseController.js:120:22)\n    at /home/sites/server/server/node_modules/parse-server/lib/Controllers/DatabaseController.js:1132:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2023-06-13T11:41:20.002Z"}
{"error":{"code":105,"message":"Invalid key name: _id"},"functionName":"getUser","level":"error","message":"Failed running cloud function getUser for user GqdXHmC08y with:\n  Input: {\"id\":\"GqdXHmC08y\"}\n  Error: {\"message\":\"Invalid key name: _id\",\"code\":105}","params":{"id":"GqdXHmC08y"},"timestamp":"2023-06-13T11:41:20.005Z","user":"GqdXHmC08y"}
{"code":105,"level":"error","message":"Invalid key name: _id","stack":"Error: Invalid key name: _id\n    at handleError (/home/sites/server/server/node_modules/parse-server/node_modules/parse/lib/node/RESTController.js:418:17)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async /home/sites/server/server/build/cloud/main.js:28:20","timestamp":"2023-06-13T11:41:20.006Z"}
{"code":105,"level":"error","message":"Invalid key name: _id","stack":"Error: Invalid key name: _id\n    at /home/sites/server/server/node_modules/parse-server/lib/Controllers/DatabaseController.js:130:13\n    at Array.forEach (<anonymous>)\n    at validateQuery (/home/sites/server/server/node_modules/parse-server/lib/Controllers/DatabaseController.js:120:22)\n    at /home/sites/server/server/node_modules/parse-server/lib/Controllers/DatabaseController.js:1132:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2023-06-13T11:49:45.238Z"}
{"error":{"code":105,"message":"Invalid key name: _id"},"functionName":"getUser","level":"error","message":"Failed running cloud function getUser for user GqdXHmC08y with:\n  Input: {\"id\":\"GqdXHmC08y\"}\n  Error: {\"message\":\"Invalid key name: _id\",\"code\":105}","params":{"id":"GqdXHmC08y"},"timestamp":"2023-06-13T11:49:45.241Z","user":"GqdXHmC08y"}
{"code":105,"level":"error","message":"Invalid key name: _id","stack":"Error: Invalid key name: _id\n    at handleError (/home/sites/server/server/node_modules/parse-server/node_modules/parse/lib/node/RESTController.js:418:17)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2023-06-13T11:49:45.244Z"}
{"code":101,"level":"error","message":"Moralis auth failed, invalid data","stack":"Error: Moralis auth failed, invalid data\n    at /home/sites/server/server/src/auth/MoralisEthAdapter.ts:32:13\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Promise.all (index 0)","timestamp":"2023-06-13T13:51:48.006Z"}
{"code":101,"level":"error","message":"Moralis auth failed, invalid data","stack":"Error: Moralis auth failed, invalid data\n    at /home/sites/server/server/src/auth/MoralisEthAdapter.ts:32:13\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Promise.all (index 0)","timestamp":"2023-06-13T13:55:17.229Z"}
